README.md

```diff
@@ -48,4 +48,4 @@ This plugin and its associated documentation is listed under the [Apache 2 licen
 
 Please visit [curity.io](https://curity.io/) for more information about the Curity Identity Server.
 
-Copyright (C) 2020 Curity AB.
```

Comment:
I think we should leave 2020, as that's when the plugin was first published, right?

Reply:
Agree, we should keep the Copyright but I'd update it to 2022.